net-p2p/qbittorrent/qbittorrent-4.6.0.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake edo multibuild systemd xdg
+
+DESCRIPTION="BitTorrent client in C++ and Qt"
+HOMEPAGE="https://www.qbittorrent.org"
+
+if [[ ${PV} == *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/qbittorrent/qBittorrent.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/qbittorrent/qBittorrent/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+	S="${WORKDIR}"/qBittorrent-release-${PV}
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+dbus +gui qt6 test webui systemd"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="|| ( gui webui )"
+
+RDEPEND="
+	>=dev-libs/openssl-1.1.1:=
+	>=net-libs/libtorrent-rasterbar-1.2.18:=
+	>=sys-libs/zlib-1.2.11
+	virtual/libiconv
+	gui? (
+		dev-libs/geoip
```

Comment:
Is this really mandatory...?

Reply:
Perhaps @xbt573 could comment/suggest something.

Reply:
zlib is used in app
geoip database downloads during runtime if i don't miss something, at least i can't find it in CMakeLists
can't find libiconv too, so i guess it's useless in ebuild

Reply:
It only seems to link to the library while compiling with `+webui`. Now I don't doubt it uses it with `+gui` too, but something you could check and verify.